HACKING.md

```diff
@@ -0,0 +1,36 @@
+Coding style Guidelines
+=======================
+
+Follow [PEP-8](https://www.python.org/dev/peps/pep-0008/) coding guidelines.
+
+
+Commit Message Guidelines
+=========================
+
+Commit Guidelines inspired by [Gnome Commit Guidelines](https://wiki.gnome.org/Git/CommitMessages).
+
+Those are only general-purpose recommended guidelines, depending on the context of each PR the following rules can vary. 
+
+Remember: the commit message is mainly for the other people, so they should be able to understand the changes made at any point in time.
+
+Example
+-------
+
+```
+short explanation of the commit
+
+Longer (optional) explanation explaining exactly what's changed and why, whether any
+external or private interfaces changed, what bugs were fixed (with bug
+tracker reference if applicable) and so forth. Be concise but not too brief. Avoid writing long lines, use newlines when necessary.
+
+[Reference to the issue solved, if any]
+```
+
+Details
+-------
+
+- First line (the brief description) must only be one sentence and should start with a lowercase letter unless it starts with a capital symbol. The message should be concise, try not to exceed 72 characters.
```

Comment:
Never seen "should start with a lowercase letter" before and I don't see how it makes sense to do so. Sentences start with an uppercase letter.